lib/vinyldns/api.rb

```diff
@@ -53,7 +53,11 @@ def self.make_request(signed_object, uri, body = '')
       url = URI(signed_object.api_url)
       https = Net::HTTP.new(url.host, url.port)
       https.use_ssl = true ? url.scheme == "https" : https.use_ssl = false
-      https.verify_mode = OpenSSL::SSL::VERIFY_NONE # SSL not signed? Don't care!
+      if ENV['VINYLDNS_VERIFY_SSL'] == false || ENV['VINYLDNS_VERIFY_SSL'] =~ /^false$/i
```

Comment:
I'd still like to see this logic in a separate method.

Reply:
👍 Specifically regarding the logic, this looks good to me & appears to work as designed.